Mathlib/Analysis/Normed/Algebra/Spectrum.lean

```diff
@@ -568,6 +568,114 @@ end CharacterSpace
 
 end WeakDual
 
+section BoundarySpectrum
+
+local notation "σ" => spectrum
+
+variable {𝕜 A SA : Type*} [NormedField 𝕜] [NormedRing A] [NormedAlgebra 𝕜 A] [CompleteSpace A]
+variable [SetLike SA A] [SubringClass SA A] [SMulMemClass SA 𝕜 A]
+variable (S : SA) [hS : IsClosed (S : Set A)] (x : S)
+
+open Topology Filter
+
+open SubalgebraClass in
+/-- Let `S` be a closed subalgebra of a Banach algebra `A`. If `a : S` is invertible in `A`,
+and for all `x : S` sufficiently close to `a` within some filter `l`, `x` is invertible in `S`,
+then `a` is invertible in `S` as well. -/
+lemma _root_.Subalgebra.isUnit_of_isUnit_val_of_eventually {l : Filter S} {a : S}
+    (ha : IsUnit (a : A)) (hla : l ≤ 𝓝 a) (hl : ∀ᶠ x in l, IsUnit x) (hl' : l.NeBot) :
```

Comment:
```suggestion
lemma Subalgebra.isUnit_of_isUnit_val_of_eventually {l : Filter S} {a : S}
    (ha : IsUnit (a : A)) (hla : l ≤ 𝓝 a) (hl : ∀ᶠ x in l, IsUnit x) (hl' : l.NeBot) :
```
We're already in the root namespace (but you probably want to put all of this in a `namespace Subalgebra` block).

Reply:
This is actually there on purpose to avoid having the namespace open to avoid collisions.

Reply:
Oh I see.